pkg/syncer/spec/spec_process_test.go

```diff
@@ -44,7 +44,6 @@ import (
 	"k8s.io/client-go/tools/clusters"
```

Comment:
please keep this in the commits that change the behaviour. Now it is very hard to understand which of the 3 other commits is related.

Reply:
I cleaned up the commit history to really have 1 change per commit, and including the related test change.

Reply:
:+1: